### DISCUSSION TITLE
Rotary axis.

### DISCUSSION THREAD

**Discussion by Hostaler** [arkypita/LaserGRBL#2353] [Ideas]
Is there a possibility that Lasergrbl can control a rotary axis?

**Comment by Tribo-2dot0** [comment]
Substitua o eixo Y pelo eixo rotativo e apenas corrija os passos/mm ($101)

**Comment by Hostaler** [comment]
Muchísimas gracias, eres muy amable.
Pero me refería a que si en el menú del programa se puede decir de alguna manera que ahora está trabajando con el rotatorio y el diámetro de la pieza a grabar para que así calcule el desarrollo de la circunferencia y los pasos.

**Reply by Tribo-2dot0** [reply]
Você pode criar um botão personalizado para isso... tente adaptar um dos botões dos rotativos da Ortur:
[Botoes Rotativo Ortur.zip](https://github.com/user-attachments/files/16847747/Botoes.Rotativo.Ortur.zip)

**Reply by Hostaler** [reply]
Los botones de Ortur son una maravilla, muchísimas gracias.
